.travis.yml

```diff
@@ -1,4 +1,5 @@
 language: c
+dist: trusty
```

Comment:
I think it has been running on `trusty` lately already but this is at least explicit